verification-annotations/src/klee.rs

```diff
@@ -105,8 +147,8 @@ pub fn expect(msg: Option<&str>) {
 macro_rules! assert {
     ($cond:expr) => {
         if ! $cond {
-            eprintln!("VERIFIER: assertion failed: {}", stringify!($cond));
-            abort();
+            eprintln!("VERIFIER: panicked at 'assertion failed: {}', {}:{}:{}", stringify!($cond), std::file!(), std::line!(), std::column!());
```

Comment:
I'm not sure "panicked" fits here

Reply:
This is a change to make verifier::assert work the same as std::assert for KLEE.
In particular, to make should_panic work, it needs to have "panicked at" in the message.

Another way to implement this change is to have this macro call 'crate::panic' and then 'crate::panic' would print the 'panicked at' part of the string

Reply:
Calling crate::panic makes it worse. I still think using "panicked" is not nice, but I guess nice is not our top priority.